### ISSUE TITLE
`pip install "pynntp[tests]"` for running the tests on other repos

### ISSUE THREAD

**Issue by cclauss** [greenbender/pynntp#56]
* https://github.com/greenbender/inn-docker
* https://github.com/cclauss/apt-get-inn2-docker
* InterNetNews/inn#313

GitHub Actions in these repos run the `pynntp` tests against an INN2 server to ensure compatibility.  `pypnp` does not ship with its tests so this currently requires:
```yaml
      - uses: actions/checkout@v4
        with:
          repository: greenbender/pynntp
      - uses: actions/setup-python@v5
        with:
          python-version: '3.x'
      - run: pip install --upgrade pip
      - run: pip install pytest
      - run: pip install --editable .
      - run: pytest
```

Would it be possible to create a `pip install "pynntp[tests]"` option that would shorten this to:
```yaml
      - uses: actions/setup-python@v5
        with:
          python-version: '3.x'
      - run: pip install --upgrade pip
      - run: pip install "pynntp[tests]" pytest
      - run: pytest
```

**Comment by greenbender** [comment]
I've looked into doing this, but it's not that straightforward.

After giving it some thought I'm also not sure it's the right approach to take for testing the docker image. By that I mean using pynntp client for testing is ok, but using the tests directly from the pynntp project is perhaps not ideal.
